scripts/illustrators_network.R

```diff
@@ -0,0 +1,310 @@
+library(readr)
+library(tidyr)
+library(reshape2)
+library(tidyverse)
+library(rebus)
+library(igraph)
+library(tidytext)
+library(optparse)
+
+tidy = function(data, authors=F){
+      data$X2 = str_remove_all(data$X2, "[,]|[.]")
+      data$X2 = str_replace_all(data$X2, "� ", "�")
+      data$X2 = str_replace_all(data$X2, " � ", "�")
+      if(authors){
+      data$X2 = str_replace_all(data$X2, "�-", "�")
+      pattern4 = or(START %R% "���� ", START %R% "����. ", START %R% "���� � �����. ", START %R% "� ��������� ")
```

Comment:
кодировка в исходном файле cp1251: Ñîñò Ñîñò è îáðàá. Â îáðàáîòêå  etc. В таком виде смержить не могу

Reply:
каждая cyrillic строка теперь аргумент функции enc2ut8() 
```suggestion
      pattern4 = or(START %R% enc2utf8("���� "), START %R% enc2utf8("����. "), START %R% enc2utf8("���� � �����. "), START %R% enc2utf8("� ��������� "))
```

Reply:
вышеупомянутая функция просто возвращает строку в кодировке UTF-8

Reply:
> каждая cyrillic строка теперь аргумент функции enc2ut8()

нет, Рома, такой подход не годится. Нужно вместо этого просто перекодировать исходный код (файл) в кодировку UTF-8 и следить за тем, чтобы входные данные тоже были в этой кодировке.